bridge/bridge.go

```diff
@@ -11,8 +11,15 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"time"
 
 	dockerapi "github.com/fsouza/go-dockerclient"
+	"github.com/rancher/go-rancher-metadata/metadata"
+)
+
+const (
+	metadataURL = "http://rancher-metadata/2015-12-19" 
```

Comment:
My apologies, I just realized that may have sounded negatively!  I did not mean it as such. :(